spec/unit_tests/moab/file_inventory_difference_spec.rb

```diff
@@ -40,7 +40,7 @@
       expect(diff_v1_v2).to be_instance_of(described_class)
     end
 
-    context 'sets attributes' do
+    describe 'attributes set' do
```

Comment:
still not in love with this, but it was a decent attempt to be clearer without a lot of effort on the individual tests ...